crates/store/re_grpc_server/src/lib.rs

```diff
@@ -27,6 +27,8 @@ use tower_http::cors::CorsLayer;
 pub const DEFAULT_SERVER_PORT: u16 = 9876;
 pub const DEFAULT_MEMORY_LIMIT: MemoryLimit = MemoryLimit::UNLIMITED;
 
+// TODO(jan): Refactor `serve`/`spawn` variants into a builder?
```

Comment:
I keep adding more of these, it's unlikely to increase any further but if it does I'll refactor it into more of a builder.